cluster/ci/config/prow/config.yaml

```diff
@@ -393,32 +393,6 @@ presubmits:
     rerun_command: "/test service-catalog"
     trigger: "((?m)^/test service-catalog,?(\\s+|$))"
 
-  - name: test_pull_request_origin_extended_conformance_azure
-    agent: jenkins
-    labels:
-      master: ci.openshift.redhat.com
-    always_run: true
```

Comment:
You can just switch this to false instead of removing them altogether.

Reply:
I didn't know what the equivalent of that was for the postsubmits and just wanted to get them disabled - does it matter?

Reply:
ok, fine by me